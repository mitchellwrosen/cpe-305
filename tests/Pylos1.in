showMove
compareMove Play at [0,0]
enterMove Play at [1,1
enterMove Play at [1,2] dammit
enterMove  Play a t [1,3]
enterMove Play   at   [  2  , 1  ]  taking [1,1] dammit
enterMove Promote from [2,2]
enterMove Play at [[2,3]
enterMove Play at [3,1]]
showMove
enterMove Play  at  [1,1]   taking  [1,2]   
showMove
saveMove Move1.pls
compareMove Play at [1,2] taking [0,0]
compareMove Promote from [1,0] to [1,1]
compareMove Play at [1,1] taking [1,2] and [0,0]

enterMove  Promote from  [1 ,1] to [1,3] taking [2,2]  and  [3,3
enterMove  Promote from  [1 ,1] to [1,3] taking [2,1]  and  [3,3]  dammit
enterMove  Promote from  [1 ,0] taking [1,1]  and  [3,3]
showMove
enterMove Promote from [1,1] to [1,2]
saveMove Move2.pls
compareMove Play at [1,1] taking [2,3]
compareMove Promote from [1,1] to [1,3]
compareMove Promote from [1,1] to [1,2]
compareMove Promote from [1,1] to [1,2] taking [0,0]
compareMove Promote from [1,1] to [1,1] taking [3,3]
enterMove Promote from [1,2] to [2,3] taking [0,3] and [1,2]
saveMove Move3.pls
loadMove Move1.pls
showMove
loadMove Move2.pls
showMove
loadMove Move3.pls
showMove

enterMove Play at [-1,0]
enterMove Play at [0,4]
showMove
quit
