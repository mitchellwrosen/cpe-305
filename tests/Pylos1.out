Play at [0, 0]

Moves are equal

Error: Bad Pylos move:  Play at [1,1

Error: Bad Pylos move:  Play at [1,2] dammit

Error: Bad Pylos move:   Play a t [1,3]

Error: Bad Pylos move:  Play   at   [  2  , 1  ]  taking [1,1] dammit

Error: Bad Pylos move:  Promote from [2,2]

Error: Bad Pylos move:  Play at [[2,3]

Error: Bad Pylos move:  Play at [3,1]]

Play at [0, 0]


Play at [1, 1] taking [1, 2]


Current move is less than entered move

Current move is less than entered move

Current move is less than entered move

Error: Bad Pylos move:   Promote from  [1 ,1] to [1,3] taking [2,2]  and  [3,3

Error: Bad Pylos move:   Promote from  [1 ,1] to [1,3] taking [2,1]  and  [3,3]  dammit

Error: Bad Pylos move:   Promote from  [1 ,0] taking [1,1]  and  [3,3]

Play at [1, 1] taking [1, 2]



Current move is greater than entered move

Current move is less than entered move

Moves are equal

Current move is less than entered move

Current move is greater than entered move




Play at [1, 1] taking [1, 2]


Promote from [1, 1] to [1, 2]


Promote from [1, 2] to [2, 3] taking [0, 3] and [1, 2]

Error: Out of bounds Pylos move:  Play at [-1,0]

Error: Out of bounds Pylos move:  Play at [0,4]

Promote from [1, 2] to [2, 3] taking [0, 3] and [1, 2]

